### ISSUE TITLE
Block Editor: Unable to Edit Text with Screen Reader

### ISSUE THREAD

**Issue by donalirl** [wordpress-mobile/gutenberg-mobile#2199]
### Context
I am opening this up on behalf of a visually-impaired user who used Apple Voiceover, but could use some help documenting the issue as I don't have an iPhone.

They report that the paragraph blocks have buttons that should allow them to edit text, but it doesn't work. They have to go back up to a more options at the top, double tap on it, then scroll all the way down to the bottom where it says 'Continue editing', and then it allows to edit the the text in the block.

### Expected behavior
They expect to be able to edit the text as soon as they tap on the paragraph block.

### Steps to reproduce the behavior
[needed]

I opened a corresponding issue in the Android app repo WordPress-Android/issues/11765 but based on this user's description, I feel the experience is very different.

**Comment by mchowning** [comment]
:wave: @SergioEstevao  and @etoledom ! Would one of you mind taking a look at this card and assigning a priority label and clarifying any steps to reproduce? I'm having quite a bit of trouble successfully editing text with Voiceover, but I'm not sure how much of that trouble is due to my lack of familiarity with Voiceover (although Voiceover seems pretty similar to Talkback) and how much is due to problems with our app. I think one of you would be better suited to assess this than me. :bow:

**Comment by etoledom** [comment]
I'd like to know the steps to reproduce. Not sure I understand the problem.
Testing this I was able to select, focus and edit paragraph blocks with VoiceOver. Didn't find any apparent problem.

Maybe there are other flows or elements that make this confusing 🤔

**Comment by mchowning** [comment]
Thanks for reporting this @donalirl ! Unfortunately, we're having some trouble recreating the issue. If you could provide us with more specific steps to reproduce the issue (or a video of the issue), that would be really helpful.

**Comment by donalirl** [comment]
Thanks for trying! It's puzzling me too so I've asked the user for more details.

**Comment by SiobhyB** [comment]
I attempted to replicate this on an _iPhone XR_, _iOS 14.6_, and _version 17.8_ of the app. 

I was able to edit a post following these steps with VoiceOver enabled:

* Double-tap through to _Blog Posts_ → _Add New_ within the app.
* Double-tap within a paragraph block in the editor and then begin typing.

This worked in the same way as the following steps, which the user seemed to indicate worked well for them:

* Double-tap the three-dot menu to the upper right.
* Double-tap on **Keep Editing** and then begin typing.

The main difference I see in the two flows is that the first requires the user to know where to tap in the editor to begin typing while the second (which works for the user) automatically gives them a position to start typing at. Perhaps this is the key? 🤔 

As others have mentioned, my lack of experience around how VoiceOver users need things to work is blocking me a bit here, and I'm only able to make guesses based on my best judgment. I do, however, feel like this is a big flaw that would have come up as part of Apple's recent accessibility review (ref: pbArwn-2d9-p2) if it was still an issue. 

I'd lean towards this no longer being an issue but, for absolute certainty, would love to either hear from the user again or for us to employ the services of someone with more expertise in assistive technologies.

**Comment by hypest** [comment]
Thanks for diving in @SiobhyB, the analysis and suggestion!

👋 @donalirl, I will close this ticket now but happy to reopen if we get more info about not working properly for users ❤️, thanks!
